docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
     volumes:
       - .:/app
       - node_modules:/app/node_modules
-    command: webpack-dev-server --hot --inline --progress --host 0.0.0.0 --config config/webpack.config.js
+    command: yarn webpack
```

Comment:
I'd use the pattern of using `yarn <command>` (without the `run`) only for the default used by npm too (`start`, `test` and others...)